tests/t223_dynamic_full.py

```diff
@@ -19,9 +19,13 @@ def pre(self):
         return TestBase.TEST_SUCCESS
 
     def build(self, name, cflags='', ldflags=''):
+        import platform
+        cflags += ' -fno-pie'
+        if not platform.machine().startswith('i686'):
+            cflags += ' -fno-plt'  # workaround of build failure
```

Comment:
I think BI (build) failure is ok to have.  It just says current environment doesn't provide such feature and we can let it fail.

Reply:
@namhyung ok, then, I'll update this when full-dynamic able.